eval/eval_validate.go

```diff
@@ -428,7 +428,7 @@ func (e *validator) validateElement(v *value, accept *schema.Schema, loc validat
 	if accept.Never {
 		return false
 	}
-	if v.unknown {
+	if v.containsUnknowns() {
```

Comment:
This one we can leave as-is. The validator understands nested unknowns.

Reply:
fixed